frontend/src/Commons/Forms/InputModules/FarmInformation/FarmInformation.tsx

```diff
@@ -58,30 +59,27 @@ const FarmInfoComponent: React.FC<InputModuleProps> = ({ farmDetails, updateFarm
     >
       <Form>
         <StyledFarmInfo>
-          <label htmlFor="FarmName">
-            <p>Name</p>
-          </label>
-          <Field
-            id="FarmName"
-            name="FarmName"
-            type="text"
-          />
-          <ErrorMessage name="FarmName" />
+          <div id="inputContainer">
+            <CustomField
+              label="Year"
+              id="Year"
+              name="Year"
+              type="number"
+              width="20%"
+            />
 
-          <label htmlFor="Year">
-            <p>Year</p>
-          </label>
-          <Field
-            id="Year"
-            name="Year"
-            type="number"
-          />
-          <ErrorMessage name="Year" />
+            <CustomField
+              label="Farm Name"
+              id="FarmName"
+              name="FarmName"
+              type="text"
+            />
+          </div>
```

Comment:
The mockup shows year before the farm name, no?

<img src="https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/60156d3c-2259-428e-8f4d-86fabfca7f6c" width=30%>

Reply:
The width is off compared to the mockup, but I'll suggest this is changed in the mockup since I feel like having 100% width for mobile might be easier to select a Field for edditing. It's currently being set to 100% on mobile and the custom width is only applied on desktop.

Reply:
I guess it does, I wonder why I was thinking that Farm name first. lol!

Reply:
Maybe because it was in the wrong order before this PR? And it also makes sense in my head that name would come before year. This could be a comment on out mockup :sweat_smile: